app/controllers/trainees_controller.rb

```diff
@@ -1,10 +1,12 @@
 class TraineesController < ApplicationController
+  before_action :authenticate
+
   def index
-    @trainees = Trainee.all
+    @trainees = current_user.trainees.all
```

Comment:
We've discussed offline that these will be updated to use Pundit scopes in https://github.com/DFE-Digital/register-trainee-teachers/pull/164 so this is fine for now.